.github/workflows/delivery-docker.yml

```diff
@@ -15,12 +20,31 @@ jobs:
         shell: bash
         run: |
           echo "::set-output name=VERSION::$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)"
-      - name: Build and Push Image
-        uses: docker/build-push-action@v1
+      - name: Setup Working Dir
+        shell: bash
+        run: |
+          cp .github/workflows/delivery/docker/buildpack.yml buildpack.yml
+      - name: Fill buildpack.yml
+        uses: cschleiden/replace-tokens@v1
+        with:
+          files: buildpack.yml
+        env:
+          PACK_VERSION: ${{ steps.version.outputs.VERSION }}
+      - name: Determine App Name
+        run: 'echo "::set-env name=IMG_NAME::$(echo ${USERNAME})/$(echo ${IMG_NAME})"'
+      - name: Pack Build/Publish
+        uses: dfreilich/pack-action@v1.0
         with:
+          args: 'build ${{ env.IMG_NAME }}:${{ steps.version.outputs.VERSION }} --builder ${{ env.BUILDER }} --publish'
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: buildpacksio/pack
-          path: .github/workflows/delivery/docker
-          build_args: VERSION=${{ steps.version.outputs.VERSION }}
-          tags: latest, ${{ steps.version.outputs.VERSION }}
+      - name: Login to Dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Tag Image as Latest
+        run: |
+          docker pull ${{ env.IMG_NAME }}:${{ steps.version.outputs.VERSION }}
+          docker tag ${{ env.IMG_NAME }}:${{ steps.version.outputs.VERSION }} ${{ env.IMG_NAME }}:latest
+          docker push ${{ env.IMG_NAME }}:latest
```

Comment:
This is relevant to #844. We currently publish the image to both tags: `latest` and `<VERSION>`, and we can only do one when we push an image. Therefore, we tag it afterwards. 

In order to access/push the image, though, we need to re-verify, because the credentials from pack aren't transferred to the rest of the workflow (it is restricted to the container that pack runs in)